### ISSUE TITLE
Change read/write theme warning dialog not converted to new style

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3176]
@doronrosenberg 

Doing the Properties 5 test plan on Safari on Mac...
1. Clone theme
2. New HTML file
3. Drag button
4. Switch theme to new theme
5. Select button
6. Change background
7. Get dialog confirming to make change to your theme... Dialog not updated to new style _AND_ noticed it mainly because I wanted to just hit enter to accept , but the link has the focus and opened docs instead of dismissing the dialog.

![GitHub Logo](http://i.imgur.com/juOY6.png)
